Test/git-issues/git-issue-2827a/A.dfy

```diff
@@ -0,0 +1,7 @@
+// RUN: %baredafny build -t:java "%s" > "%t"
+// RUN: java -jar A.jar >>  "%t"
+// RUN: %diff "%s.expect" "%t"
+
+method Main() {
```

Comment:
Do we have any tests anywhere for something like?
```dafny
method {:main} MyMainThing
```

Maybe not in scope here, but just wondering

Reply:
There are such tests, yes -- and for a variety of error situations (more than one Main or {:main} or inappropriate Main)